bin/anvi-run-cazymes

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+
+import sys
+
+import anvio
+import anvio.cazymes as cazymes
+
+from anvio.errors import ConfigError, FilesNPathsError
+from anvio.terminal import time_program
+
+__author__ = "Developers of anvi'o (see AUTHORS.txt)"
+__copyright__ = "Copyleft 2015-2020, the Meren Lab (http://merenlab.org/)"
+__license__ = "GPL 3.0"
+__version__ = anvio.__version__
+__authors__ = ['mschecht']
+__requires__ = ['contigs-db', 'cazyme-data']
+__provides__ = ['functions']
+__description__ = "Run dbCAN CAZymes on contigs-db"
+
+
+@time_program
+def main(args):
+    p = cazymes.CAZyme(args)
+    p.process()
+
+
+if __name__ == '__main__':
+    from anvio.argparse import ArgumentParser
+
+    parser = ArgumentParser(description=__description__)
+
+    parser.add_argument(*anvio.A('contigs-db'), **anvio.K('contigs-db'))
+    parser.add_argument('--noise-cutoff-terms', default=None, help="Choose a threshold for the CAZyme HMMs e.g. --noise-cutoff-terms \"-E 1\". \
+                        If you are not sure what to pick run `hmmsearch -h` or `hmmscan -h` and check out your options.")
```

Comment:
is `-E 1` a good idea as the default cutoff? I feel like we could be more stringent because I'm afraid no one will pay attention, and we will make people include too much garbage in their hits due to our defaults :(

Reply:
At the moment there is no default cutoff, but I like the idea of making a stringent default. For now I will make the default:  `-E 1e-12`